resources/destination.rb

```diff
@@ -76,6 +78,8 @@
 end
 
 action :create do
+  dest = source_dest_config_builder(driver: new_resource.driver, path: new_resource.path, parameters: new_resource.parameters, configuration: new_resource.configuration)
```

Comment:
This could do with being multiline for readability 👍🏼

Reply:
This might get reverted as I've got a feeling this might be a Chef 17 change/bug as I had it multi line before straight in the template variables but it raises a wrong number of arguments error like that even though (afaik) it shouldn't.

Reply:
Well i've bashed my head against this for most of the morning and still can't get it to not fall over with `wrong number of arguments (given 1, expected 0)` unless it's outside of the block. It's **got** to be a Ruby 3 thing but I can't see why as it doesn't catch any of the function arg deprecations.

So indented and go with that for the time being.